### PR TITLE
New alpha items

### PR DESCRIPTION
**Pull request by nopothegamer**
d leg

### CODE REVIEW

**Comment by nopothegamer** [comment]
> d leg

when did i type this?????

**Comment by 8KCoffeeWizard** [comment]
nopo the game carrying repo

File: constants/petnums.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "GOLDEN_DRAGON": {
-    "COMMON": {
+    "LEGENDARY": {
```
Comment (IRONM00N) [inline constants/petnums.json:3]: Since when are there multiple types of dragon pets

**Review by IRONM00N — APPROVED** [review]
Looks good

File: constants/petnums.json (hunk 1 above)
Comment (nopothegamer) [reply constants/petnums.json:3]: there wasnt i just did a lil brain fart

**Pull request merged**